server/services/assignment_service.go

```diff
@@ -3,34 +3,48 @@ package services
 import (
 	"github.com/bokoness/lashon/database"
 	"github.com/bokoness/lashon/models"
+	"github.com/gofiber/fiber/v2/log"
 )
 
-func GetAssignments(assignmentId string) []models.Assignment {
-	var assignments []models.Assignment
+func GetAssignments(assignmentId string) ([]*models.Assignment, error) {
```

Comment:
Study question
Why use a pointer here?

Reply:
So it will be possible to return `nil` when needed